docs/ai/quickstarts/prompt-model.md

```diff
@@ -90,8 +89,8 @@ Complete the following steps to create a .NET console app to connect to an AI mo
 
    ```bash
    dotnet user-secrets init
-   dotnet user-secrets set OpenAIKey <your-openai-key>
```

Comment:
I thought it was customary to have lowercase placeholders? That could just be me.... And I probably don't even follow that advice... 😁

Reply:
I don't think the guidance says anything about casing. [This](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/formatting-text-in-instructions#in-documentation-and-technical-content) just says to use angle brackets.